src/internal/syscall/windows/syscall_windows.go

```diff
@@ -98,6 +101,20 @@ type IpAdapterPrefix struct {
 	PrefixLength uint32
 }
 
+type IpAdapterWinsServerAddress struct {
+	Length   uint32
+	Reserved uint32
+	Next     *IpAdapterWinsServerAddress
+	Address  SocketAddress
+}
+
+type IpAdapterGatewayAddress struct {
```

Comment:
Let me know if I should add a documentation link for it, for example: https://learn.microsoft.com/en-us/windows/win32/api/iptypes/ns-iptypes-ip_adapter_gateway_address_lh
I did not to conform with existing structures not having such documentation link.

That would had helped me to understand the structures there, although I managed without initially.